index.html

```diff
@@ -1677,19 +1673,15 @@
                 </li>
               </ol>
             </li>
-            <li>If the <a>presentation availability promise</a> for
-            <code>presentationRequest</code> is not <code>null</code>, return
-            the <code>presentationRequest</code> object's <a>presentation
-            availability promise</a> and abort these steps.
+            <li>If there is an unsettled <a>Promise</a> from a previous call to
```

Comment:
If the returned Promise is settled, this will repeat all the steps below, generating a new Availability object, which is what we were trying to avoid.  Drop "unsettled" here?